### PR TITLE
Add `work_pool_name` to work queue API responses

### PR DESCRIPTION
**Pull request by desertaxle**
Work pools can only be retrieved via their name, so if reading work queues directly, like in https://github.com/PrefectHQ/prefect-ui-library/pull/1428, an extra call is required to determine the name of the work pool that a work queue resides in. This PR adds `work_pool_name` to the response for work queue API routes to make it easier to get work pool information when starting with a work queue.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

**Comment by zhen0** [comment]
Thank you @desertaxle 🌟

**Comment by zhen0** [comment]
@desertaxle - does this need anything or shall we find a reviewer?

File: src/prefect/client/orchestration.py [hunk 1]
```diff
@@ -777,7 +776,7 @@ async def create_work_queue(
         concurrency_limit: Optional[int] = None,
         priority: Optional[int] = None,
         work_pool_name: Optional[str] = None,
-    ) -> schemas.core.WorkQueue:
+    ) -> WorkQueue:
```
Comment (chrisguidry) [inline src/prefect/client/orchestration.py:779]: Thank you, I prefer the legibility of omitting the namespaces :bow:

**Review by chrisguidry — APPROVED** [review]
This all makes sense to me!

**Pull request merged**